contrib/omhttp/omhttp.c

```diff
@@ -144,27 +183,35 @@ static struct cnfparamdescr actpdescr[] = {
 	{ "server", eCmdHdlrArray, 0 },
 	{ "serverport", eCmdHdlrInt, 0 },
 	{ "healthchecktimeout", eCmdHdlrInt, 0 },
+	{ "httpcontenttype", eCmdHdlrGetWord, 0 },
 	{ "httpheaderkey", eCmdHdlrGetWord, 0 },
 	{ "httpheadervalue", eCmdHdlrGetWord, 0 },
 	{ "uid", eCmdHdlrGetWord, 0 },
 	{ "pwd", eCmdHdlrGetWord, 0 },
 	{ "restpath", eCmdHdlrGetWord, 0 },
 	{ "dynrestpath", eCmdHdlrBinary, 0 },
-	{ "bulkmode", eCmdHdlrBinary, 0 },
+	{ "bulkmode", eCmdHdlrBinary, 0 }, /* batch and bulkmode are synonyms */
+	{ "batch", eCmdHdlrBinary, 0 },
```

Comment:
why not stick with a single param? Two synonyms alwas cause trouble in the long term... Just a general thought.

Reply:
Wanted to change terminology to be more general, since `bulkmode` is an elasticsearch-specific concept, while still being backward-compatible. Introduced the `batch.format`, `batch.maxbytes`, `batch.maxsize` here as well.

Agree that synonyms may be problematic long term. Would you prefer to keep the previous parameter names or drop them in favor of more nicely namespaced params?